docs/running-your-own-kernel.md

```
@@ -20,7 +20,7 @@ The best way to run a custom kernel is to find an existing image, for example on
 
 ## R kernel
 
-In the right sidebar, pick your preferred version of R from the dropdown in the environment section. We recommended choosing the `deepnote/ir_with_libs` image. This will install R 4.2 and many common data science libraries for you (see the image details [here](https://hub.docker.com/r/deepnote/ir-with-libs)).
+In the right sidebar, pick your preferred version of R from the dropdown in the environment section. We recommend choosing the `deepnote/ir-with-libs:4.2.0` image. This installs R 4.2.0 and many common data science libraries for you (see the image details [here](https://hub.docker.com/r/deepnote/ir-with-libs)).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)